src/rv/comm/drawing/commands/Control.java

```diff
@@ -13,7 +14,17 @@
     public Control(ByteBuffer buf, Viewer viewer) {
         super();
         this.viewer = viewer;
-        agent = Command.readAgent(buf, viewer.getWorldModel());
+
+	int type = ByteUtil.uValue(buf.get());
```

Comment:
I guess this line is the one that actually fixes the issue since it increments the byte position? I thought we could get away with not having a switch here at all, similar to https://github.com/Gama11/RoboViz/blob/master/src/rv/comm/drawing/commands/DrawOption.java that also only has one command, guess not. :)

Also, I suppose it's preferable to print an error instead of ignoring the type like `DrawOption` does from a user perspective - maybe we should print an error there as well?